From b1fa980d86b8be16ed5e1b7be22ff6def651d558 Mon Sep 17 00:00:00 2001
From: Nadav Haklai <nadavh@marvell.com>
Date: Wed, 24 Oct 2012 10:40:44 +0200
Subject: [PATCH 0288/1825] Fix sysfs command to read/write from PHY
 registers. All arguments for all commands in hex
 format

https://github.com/MISL-EBU-System-SW/misl-windriver.git linux-3.4.69-14t2-read
commit 509a36a7c42b6932d8fec87a486f8dbef3859772

Change-Id: I47b4e63139a5c36592f5dbf52107513aee7ab567
Signed-off-by: Nadav Haklai <nadavh@marvell.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 arch/arm/plat-armada/mv_hal/eth-phy/mvEthPhy.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/plat-armada/mv_hal/eth-phy/mvEthPhy.c b/arch/arm/plat-armada/mv_hal/eth-phy/mvEthPhy.c
index 30bc00c..3a91f7d 100644
--- a/arch/arm/plat-armada/mv_hal/eth-phy/mvEthPhy.c
+++ b/arch/arm/plat-armada/mv_hal/eth-phy/mvEthPhy.c
@@ -280,7 +280,7 @@ MV_STATUS mvEthPhyRegPrint(MV_U32 phyAddr, MV_U32 regOffs)
 
 	status = mvEthPhyRegRead(phyAddr, regOffs, &data);
 	if (status == MV_OK)
-		mvOsPrintf("reg=%d: 0x%04x\n", regOffs, data);
+		mvOsPrintf("phy=0x%x, reg=0x%x: 0x%04x\n", phyAddr, regOffs, data);
 	else
 		mvOsPrintf("Read failed\n");
 
-- 
1.7.5.4

